rets/client/decoder.py

```diff
@@ -107,4 +107,5 @@ def _decode_date(value: str, include_tz: bool) -> datetime:
     'Int': int,
     'Long': int,
     'Decimal': Decimal,
+    'Timestamp': datetime.now()
```

Comment:
Won't this hard-code it to the date the Python process starts instead of the current time. Should this be a lambda returning `datetime.now()` instead? Or should it be `utcnow()`?

Reply:
Hey,
If it will be utcnow() it's okay. just consider that when my query is 'LastUpdated=2015-09-01T00:00:00Z' _get_decoder method returns exception about unknown data type, it should be fixed because some other MLS servers use timestamp as well.